odfuzz/config.py

```diff
@@ -2,7 +2,21 @@
 
 import os
 
-from odfuzz.constants import ENV_ODFUZZ_CERTIFICATE_PATH, DEFAULT_ASYNC_REQUESTS_NUM, DEFAULT_SAP_CLIENT, DEFAULT_DATA_FORMAT, DEFAULT_URLS_PER_PROPERTY, ENV_SAP_CLIENT, ENV_DATA_FORMAT, ENV_URLS_PER_PROPERTY, ENV_ASYNC_REQUESTS_NUM
+from odfuzz.constants import (
```

Comment:
even tho it is definitely better, it is good practice to not mix PR containing fix or feature with changes that are formatting only. This should ideally be different PR.